scripts/DeployNode.sh

```diff
@@ -1,26 +1,35 @@
+#!/usr/bin/env bash
+
+# Usage:
+# ./DeployNode.sh [GCP machine type, defaults to n1-highmem-8]
+#
+# Example:
+# ./DeployNode.sh n1-standard-4
+
 # This script has the following preconditions:
-# 1. You have a GCE SSD disk called node-disk with enough space for a parity db.
+# 1. You have a GCE SSD disk called node-disk with enough space for a geth db.
 # 2. node-disk is writable by most/all users (rw filesystem permissions have been granted).
-# 3. node-disk contains a folder called io.parity.ethereum (can be empty).
+# 3. node-disk contains a folder called ethereum (can be empty).
+
+MACHINE_TYPE=$1
+: ${MACHINE_TYPE:=n1-highmem-8}
```

Comment:
this might be overly big 
![image](https://user-images.githubusercontent.com/4429761/90023129-5ac56800-dc81-11ea-9039-8f9732e74698.png)
i feel like highmem-2 is probably ok and bump it up if needed. what do you think?

Reply:
sorry just saw your note in desc. if you are good with that, i am

Reply:
I based it on this post by the geth team: https://blog.ethereum.org/2019/07/10/geth-v1-9-0/ where they benchmarked syncing (this was roughly the size they used for `fast` mode).

Reply:
Will add this to the desc.